CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - Now properly defaulting function field to empty string when converting from 
   model to state model to preserve backwards compatibility.
+### Added
```

Comment:
The release process for this does not automatically update the released dates. Please update the above change to version 0.20.3 and add the data from the merge. Then put your addition under a new unreleased section

Reply:
updated